config/application.rb

```diff
@@ -10,7 +10,7 @@
 require "action_mailer/railtie"
 require "action_view/railtie"
 require "action_cable/engine"
- require "sprockets/railtie"
+# require "sprockets/railtie"
```

Comment:
This is the line that was causing our heroku issues, without it, things should run smoothly.

Reply:
Awesome! Great work catching that!

Reply:
yes nice work here!